documentation/src/main/wiki/Server.md

```diff
@@ -214,6 +215,15 @@ The `repositories`, `splitter` and `assembler` verticles are specific to the def
 | `address`      | `String`         | &#10004;       | Event bus address of the **Knot** verticle |
 | `onTransition` | `KnotRouteEntry` |        | Describes routing to addresses of other Knots based on the transition trigger returned from current Knot.<br/>`"onTransition": { "go-d": {}, "go-e": {} }` |
 
+
+### AccessLogConfiguration options
+| Name  | Type  | Mandatory | Description  |
+|-------:|:-------:|:-------:  |-------|
+| `enabled`   | `boolean` |       | Enable/Disable access log. Default is `true` |
+| `immediate` | `boolean` |       | Log before request or after. Default is `false` - log after request |
+| `format`    | `String` |        | Format of the access log. Allowed valueds are `DEFAULT`, `SHORT`, `TINY`. Default tries to log in a format similar to Apache log format, while the other 2 are more suited to development mode. Default format is `DEFAULT` |
```

Comment:
Can you provide here any url to documentation defining `Apache log format`?
Maybe there should be a short note with link to `#configure-access-log` where those formats are explained?